File: app/src/main/java/com/pronab/sunbank/api/Service.java
package com.pronab.sunbank.api;

import com.google.gson.JsonArray;

import retrofit2.Call;
import retrofit2.http.GET;

/**
 * Created by delaroy on 7/3/17.
 */

public interface Service {

    @GET("/transactions")
    Call<JsonArray> readTranArray();

}

File: app/src/main/java/com/pronab/sunbank/api/Service.kt
package com.pronab.sunbank.api

import com.google.gson.JsonArray

import retrofit2.Call
import retrofit2.http.GET

/**
 * Created by delaroy on 7/3/17.
 */

interface Service {

    @GET("/transactions")
    fun readTranArray(): Call<JsonArray>

}
